pkgs/development/python-modules/mizani/default.nix

```diff
@@ -1,15 +1,20 @@
-{ buildPythonPackage
+{ lib
+, buildPythonPackage
 , fetchFromGitHub
-, lib
 , matplotlib
 , palettable
 , pandas
 , pytestCheckHook
+, pythonOlder
+, scipy
```

Comment:
nit: alphabetize

Reply:
`lib` and `stdenv` are usually (or often) mentioned first. I can revert that as it's the primary maintainer's call.